gammapy/background/reflected.py

```diff
@@ -188,7 +191,7 @@ def setup(self):
         )
 
         # Distance image
-        self._distance_image = _compute_distance_image(self.exclusion_mask)
+        self._distance_image = _compute_distance_image(self.reference_map)#exclusion_mask)
```

Comment:
Here's a meaningless comment at the end of the line...

Reply:
corrected